web/tsconfig.json

```diff
@@ -9,14 +9,9 @@
     "jsx": "react",
     "allowSyntheticDefaultImports": true,
     "paths": {
-      "@/*": ["src/*"],
-      "@assets/*": ["src/assets/*"],
-      "@client": ["src/client/index.js"],
-      "@client/*": ["src/client/*"],
-      "@components/*": ["src/components/*"],
-      "@context/*": ["src/context/*"],
-      "@lib/*": ["src/lib/*"],
-      "~icons/*": ["node_modules/@material-symbols/svg-400/outlined/*"]
+      "~/*": ["src/*"],
+      "~/client": ["src/client/index.js"],
+      "@icons/*": ["node_modules/@material-symbols/svg-400/outlined/*"]
```

Comment:
Use the @ here because this alias actually is a shortcut for an import from a node package.